libs/blocks/review/review.js

```diff
@@ -54,9 +55,6 @@ const App = ({ rootEl, strings }) => html`
       visitorId=${getVisitorId()}
       reviewPath=${getReviewPath(strings.postUrl)}
       initialValue=${strings.initialValue}
-      onRatingSet=${({ rating, comment }) => {}}
-      onRatingHover=${({ rating }) => {}}
-      onReviewLoad=${({ hasRated, rating }) => {}}
```

Comment:
If we remove those and they are never used, should we also remove them from the rest of the children?

Reply:
Thank you @mokimo @nkthakur48
I fixed eslint by commenting out argument that are not used because I am not sure if these props are planed for some future usage.
If you think they should be removed, I can do that right away.

Reply:
If the callbacks will be used in the future we can always check the code history and add them. Please remove them if they're not used.